Test_design.md

```diff
@@ -0,0 +1,64 @@
+# Test design for selected functionalities of the website pracuj.pl

```

Comment:
Jeśli używasz polskiego dalej, to czemu tu jest po angielsku? Zdecyduj się na 1 język i bądź konsekwentny. Jeśli chcesz używać tego repo jako wizytówki to proponuje wszystko pisać po angielsku.